requirements.txt

```diff
@@ -1,5 +1,6 @@
 black==21.7b0
 mypy==0.931
 pytest
+pytest-cov
 pytest-timeout
-py-algorand-sdk
+git+https://github.com/algorand/py-algorand-sdk@teal-blackbox
```

Comment:
This PR is blocked by [PySDK's #298](https://github.com/algorand/py-algorand-sdk/pull/298)